install/gcp/deployment_manager/istio-cluster.jinja

```diff
@@ -129,12 +142,11 @@ resources:
           helm template install/kubernetes/helm/istio --name istio --namespace istio-system $ISTIO_OPTIONS  > istio.yaml
 
           kubectl apply -f istio.yaml
-          kubectl label namespace default istio-injection=enabled
 
           sleep 150
 
           kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
           kubectl apply -f samples/bookinfo/networking/bookinfo-gateway.yaml
 
           gcloud beta runtime-config configs variables set success/{{ CLUSTER_NAME }}-waiter success --config-name $(ref.{{ CLUSTER_NAME }}-config.name)
-          gcloud -q compute instances delete {{ CLUSTER_NAME }}-vm --zone {{ properties['zone'] }}
+          #gcloud -q compute instances delete {{ CLUSTER_NAME }}-vm --zone {{ properties['zone'] }}
```

Comment:
why is this commented out?

Reply:
inadvertently left in for debugging; uncommented now;